src/herbpy/herbrobot.py

```diff
@@ -271,7 +270,72 @@ def SetStiffness(self, stiffness):
         self.head.SetStiffness(stiffness)
         self.left_arm.SetStiffness(stiffness)
         self.right_arm.SetStiffness(stiffness)
-
+        
+    def DetectHuman(self, env, orhuman=2, load_hum=True, 
+                    hrc=False, continuous=True,      
+                    herb_sim=True, enable_legs=True,             
+                    segway_sim=True, action='stamp'):
```

Comment:
Please an a docstring for each argument. Most functions in PrPy are documented properly, so you can look there for examples.